dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -566,6 +592,38 @@ export const FrontSection = ({
 								showDateHeader={showDateHeader}
 								editionId={editionId}
 							/>
+							{!isOnPaidContentFront &&
+								!!editionBranding &&
+								editionBranding?.branding &&
+								index === 0 && (
```

Comment:
Here we're trying to re-create [this logic](https://github.com/guardian/frontend/blob/main/facia/app/views/fragments/frontBody.scala.html#L44) from frontend:

Reply:
We are not explicitly checking if it's a thrasher because [we are not using `FrontSection` for thrashers](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/layouts/FrontLayout.tsx#L347C55-L375)